actors.md

```diff
@@ -658,23 +662,23 @@ func SubmitPost(proofs PoStProof, faults [FaultSet], recovered Bitfield, done Bi
 
 	feesRequired := 0
 
-	if chain.Now() > miner.ProvingPeriodEnd+GenerationAttackTime(miner.SectorSize) {
+	if chain.Now() > self.ProvingPeriodEnd+GenerationAttackTime(self.SectorSize) {
         // slashing ourselves
         SlashStorageFault(self)
         return
-	} else if chain.Now() > miner.ProvingPeriodEnd {
-		feesRequired += ComputeLateFee(miner.Power, chain.Now()-miner.ProvingPeriodEnd)
+	} else if chain.Now() > self.ProvingPeriodEnd {
+		feesRequired += ComputeLateFee(miner.power, chain.Now()-self.provingPeriodEnd)
 	}
 
-	feesRequired += ComputeTemporarySectorFailureFee(miner.SectorSize, recovered)
+	feesRequired += ComputeTemporarySectorFailureFee(self.sectorSize, recovered)
```

Comment:
Is `ComputeTemporarySectorFailureFee()` going to be defined in @dignifiedquire's faults PR?

Reply:
it should be, yes